app/src/organisms/RobotSettingsCalibration/CalibrationDataDownload.tsx

```diff
@@ -67,27 +80,65 @@ export function CalibrationDataDownload({
   }
 
   return (
-    <Box paddingBottom={SPACING.spacing5}>
-      <Flex alignItems={ALIGN_CENTER} justifyContent={JUSTIFY_SPACE_BETWEEN}>
-        <Box marginRight={SPACING.spacing6}>
-          <Box css={TYPOGRAPHY.h3SemiBold} marginBottom={SPACING.spacing3}>
-            {t('about_calibration_title')}
-          </Box>
-          <StyledText as="p" marginBottom={SPACING.spacing3}>
-            {t('about_calibration_description')}
-          </StyledText>
-          <Link
-            role="button"
-            css={TYPOGRAPHY.linkPSemiBold}
-            onClick={() => setShowHowCalibrationWorksModal(true)}
+    <>
+      {enableCalibrationWizards ? (
+        <Box paddingTop={SPACING.spacing5} paddingBottom={SPACING.spacing2}>
+          <Flex
+            alignItems={ALIGN_CENTER}
+            justifyContent={JUSTIFY_SPACE_BETWEEN}
+          >
+            <Box marginRight={SPACING.spacing6}>
+              <Box css={TYPOGRAPHY.h3SemiBold} marginBottom={SPACING.spacing3}>
+                {t('robot_calibration:download_calibration_title')}
+              </Box>
+              <StyledText as="p" marginBottom={SPACING.spacing3}>
+                {t(
+                  downloadIsPossible
+                    ? 'robot_calibration:download_calibration_available'
+                    : 'robot_calibration:download_calibration_unavailable'
+                )}
+              </StyledText>
+            </Box>
+            <TertiaryButton
+              onClick={onClickSaveAs}
+              disabled={!downloadIsPossible}
+            >
+              <Icon
+                name="download"
+                width="0.625rem"
```

Comment:
using `width` totally makes sense but we use `size` especially when we want to use the same value to `width` and `height`.
 
```suggestion
                size="0.625rem"
```

Reply:
From Figma the size would be 12px x 12px (0.75rem)
![Screenshot 2022-12-16 at 7 08 46 PM](https://user-images.githubusercontent.com/474225/208211178-2f1b347e-38f9-4bd8-b904-dbb6baf95ece.png)

Reply:
Ah, ok. I was looking at the "Union" (which you get when you double-click on the icon). That has it as 0.625rem wide and *not* quite square. 
![Screenshot 2022-12-16 at 11 50 17 PM](https://user-images.githubusercontent.com/2960/208225740-997bd946-38c3-4c50-9c40-fe53f64e0dc2.png)